etcdmain/etcd.go

```diff
@@ -321,7 +321,7 @@ func startEtcd(cfg *config) (<-chan struct{}, error) {
 	if cfg.v3demo {
 		// set up v3 demo rpc
 		grpcServer := grpc.NewServer()
-		etcdserverpb.RegisterKVServer(grpcServer, v3rpc.New(s))
+		etcdserverpb.RegisterKVServer(grpcServer, v3rpc.NewKV(s))
```

Comment:
NewKVServer? This name sounds natural when reading `NewWatchServer` below, and `v3rpc` has no hint for the server.